.circleci/config.yml

```diff
@@ -492,7 +492,7 @@ jobs:
           command: docker-compose logs -f
           background: true
       - run:
-          command: ./scripts/ddtest scripts/run-tox-scenario '^django_contrib-'
+          command: DD_TRACE_AGENT_URL=http://localhost:8127 ./scripts/ddtest scripts/run-tox-scenario '^django_contrib-'
```

Comment:
should have a `run_tox_scenario` like command for this, then we can set the env variable in one place "globally"?

Reply:
we can set it once as a project level global variable, but that seems like we are just hiding the configuration in a place we'll forget about.

Reply:
Good idea